docs/man/futhark.rst

```diff
@@ -66,6 +66,12 @@ A Futhark compiler development command, intentionally undocumented and
 intended for use in developing the Futhark compiler, not for
 programmers writing in Futhark.
 
+futhark eval [-f FILE] [-w] <exprs...>
+----------------------------------
```

Comment:
Add more dashes or Sphinx will complain.

Reply:
pleased the sphinx